<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Gimnasio</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBienvenida">
    <property name="title">
     <string>Bienvenida</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuClientes">
    <property name="title">
     <string>Clientes</string>
    </property>
    <addaction name="actionBuscarCliente"/>
    <addaction name="actionRegistrar"/>
   </widget>
   <addaction name="menuBienvenida"/>
   <addaction name="menuClientes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionBuscarCliente">
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="statusTip">
    <string>Busqueda de clientes por nombre</string>
   </property>
  </action>
  <action name="actionRegistrar">
   <property name="text">
    <string>Registrar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
